src/main/java/com/jaspersoft/jasperserver/jaxrs/client/apiadapters/bundles/BundlesService.java

```diff
@@ -71,7 +73,11 @@ public BundlesService forLocale(Locale locale) {
         JerseyRequest<Map<String, Map<String, String>>> request =
                 JerseyRequest.buildRequest(sessionStorage, new GenericType<Map<String, Map<String, String>>>() {
                 }, new String[]{"/bundles"}, new DefaultErrorHandler());
-        request.setAccept(MediaType.APPLICATION_JSON).addHeader("Accept-Language", locale.toString().replace('_', '-'));
+
+        request.setAccept(MediaType.APPLICATION_JSON);
+        if (!"".equals(this.locale.getLanguage())) {
```

Comment:
This method will throw NuppPointerException after somebody called forLocale((Locale)null)